R/FilteredData.R

```diff
@@ -52,13 +57,43 @@
 #' isolate(datasets$get_filter_state())
 #' isolate(datasets$get_call("iris"))
 #' isolate(datasets$get_call("mtcars"))
+#' @examplesIf requireNamespace("MultiAssayExperiment")
+#' ### set_filter_state
+#'
+#' utils::data(miniACC, package = "MultiAssayExperiment")
+#' datasets <- FilteredData$new(list(iris = iris, mae = miniACC))
+#' fs <-
+#'   teal_slices(
+#'     teal_slice(
+#'       dataname = "iris", varname = "Sepal.Length", selected = c(5.1, 6.4),
+#'       keep_na = TRUE, keep_inf = FALSE
+#'     ),
+#'     teal_slice(
+#'       dataname = "iris", varname = "Species", selected = c("setosa", "versicolor"),
+#'       keep_na = FALSE
+#'     ),
+#'     teal_slice(
+#'       dataname = "mae", varname = "years_to_birth", selected = c(30, 50),
+#'       keep_na = TRUE, keep_inf = FALSE
+#'     ),
+#'     teal_slice(dataname = "mae", varname = "vital_status", selected = "1", keep_na = FALSE),
+#'     teal_slice(dataname = "mae", varname = "gender", selected = "female", keep_na = TRUE),
+#'     teal_slice(
+#'       dataname = "mae", varname = "ARRAY_TYPE",
+#'       selected = "", keep_na = TRUE, datalabel = "RPPAArray", arg = "subset"
+#'     )
```

Comment:
this gave me a warning
```
filters for columns: ARRAY_TYPE excluded from mae
```

is this intended? Can we make it a warning-fee example?

Reply:
Same for example in `MAEFilteredDataset.Rd`

Reply:
This is a reoccurring problem with how `excluded_variables` (or whatever the name) is set.
The only person who can fix this quickly is @gogonzo.

Reply:
Started a separate issue for that https://github.com/insightsengineering/teal.slice/issues/531